### ISSUE TITLE
Is placeOrder endpoint working?

### ISSUE THREAD

**Issue by theHour** [bithumb-pro/bithumb.pro-official-api-docs#64]
Hi, 

I am playing around and I'm trying to place an order via REST API, however I am not able as I'm constantly getting 9005 error from the server.

Pretty much every other endpoint works except this one.

Is this API ENDPOINT in use?

edit:
Adding a example for a request params:

`  const params = {
    "apiKey": API_KEY,
    "msgNo": 'msgNo${Date.now()}',
    "price": "3.65",
    "quantity": "1",
    "side": "buy",
    "symbol": "DYP-USDT",
    "timestamp": Date.now(),
    "type": "limit"
  }`
  
  Basically I'm trying to buy DYP for USDT. Am I doing this correctly or am I missing something?
  P.S. Signature is added later and it works correctly for every other method.

**Comment by theHour** [comment]
@godlikers Any help is appreciated.

**Comment by theHour** [comment]
Please ignore this, I've made a mistake in endpoints url!
Works as expected.
